linode/manager
# fix: [M3-8274] - Fix parsing issue causing in Kubernetes Version field

## Description 📝
This PR handles decimal value correctly in Kubernetes Version field (LKE Create Page).

## Target release date 🗓️
6/24


| Before  | After   |
| ------- | ------- |
|![image](https://github.com/linode/manager/assets/119517080/7ca33214-c3b0-4222-a479-6b517957cc43) |![image](https://github.com/linode/manager/assets/119517080/532fca49-6275-4bbe-beed-c88b4e134a78) |

## How to test 🧪

### Reproduction steps
(How to reproduce the issue, if applicable)
- Navigate to LKE create page in dev
- Notice latest version of Kubernetes Version field. - missing decimal number

### Verification steps
(How to verify changes)
- Checkout this branch
- Navigate to LKE create page
- Notice latest version of Kubernetes Version field. - renders decimal number correctly

## As an Author I have considered 🤔

*Check all that apply*
